app/assets/stylesheets/welcome.scss

```diff
@@ -1,29 +1,34 @@
-// .vidcontainer {
-//   position: absolute;
-//   width: 100%;
-//   top: 50%;
-//   left: 50%;
-//   margin: -100px 0 0 0;
-// }
+.vidcontainer {
+  position: absolute;
+  width: 100%;
+  top: 50%;
+  left: 50%;
+  margin: 0 0 0 0;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `0` instead of `0 0 0 0`